data/assets/css/components/_event-summary.scss

```diff
@@ -32,22 +37,36 @@
     text-transform: uppercase;
 
     .date-badge {
-      position: absolute;
-      top: 20px;
-      right: -10px;
-      background-color: $brand-color;
-      padding: 0 5px;
-      z-index: 1;
-
-      &.invert {
-        color: $brand-color;
-        background-color: $black;
+      display: none;
+
+      @include size-medium {
+        display: block;
+        position: absolute;
+        top: 20px;
+        right: -10px;
+        background-color: $brand-color;
+        padding: 0 5px;
+        z-index: 1;
+
+        &.invert {
+          color: $brand-color;
+          background-color: $black;
+        }
       }
     }
 
     .reset-link {
       margin-bottom: $space-small;
       background-image: none;
+      float: left;
```

Comment:
could you use flex instead of float?

Reply:
I'll work on that this week. I think because of the nesting, `float` was the easier solution, but I'll see if we can use flexbox instead.